src/plugins/extensions/associatedAuthority/fields.js

```diff
@@ -0,0 +1,776 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+    CompoundInput,
+    TextInput,
+    TermPickerInput,
+    StructuredDateInput,
+  } = configContext.inputComponents;
+
+  const {
+    DATA_TYPE_STRUCTURED_DATE,
+  } = configContext.dataTypes;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    extensions,
+  } = configContext.config;
+
+  return {
+    personGroupList: {
+      [config]: {
+        view: {
+          type: CompoundInput,
+        },
+      },
+      personGroup: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.ext.associatedAuthority.personGroup.name',
+              defaultMessage: 'Person',
+            },
+          }),
+          repeating: true,
+          view: {
+            type: CompoundInput,
+          },
+        },
+        person: {
+          [config]: {
+            messages: defineMessages({
+              fullName: {
+                id: 'field.ext.associatedAuthority.person.fullName',
+                defaultMessage: 'Person associated',
+              },
+              name: {
+                id: 'field.ext.associatedAuthority.person.name',
+                defaultMessage: 'Associated',
+              },
+            }),
+            view: {
+              type: AutocompleteInput,
+              props: {
+                source: 'person/local,person/ulan',
+              },
+            },
+          },
+        },
+        personType: {
+          [config]: {
+            messages: defineMessages({
+              fullName: {
+                id: 'field.ext.associatedAuthority.personType.fullName',
+                defaultMessage: 'Person relationship/type',
+              },
+              name: {
+                id: 'field.ext.associatedAuthority.personType.name',
+                defaultMessage: 'Relationship/Type',
+              },
+            }),
+            view: {
+              type: TermPickerInput,
+              props: {
+                source: 'chronologypersonrelations',
```

Comment:
Hmm, are these relation terms really going to be specific to each record type? If so, we can't really reuse this config, and it shouldn't be an extension -- which would be bad. We should clarify this.